libnavui-maps/src/main/java/com/mapbox/navigation/ui/maps/guidance/signboard/model/MapboxExternalFileResolver.kt

```diff
@@ -0,0 +1,145 @@
+
+        try {
+            return getTypeface(fontFamily, fontWeight, fontStyle, ".ttf")
+        } catch (exception: RuntimeException) {
+        }
+
+        return try {
+            getTypeface(fontFamily, fontWeight, fontStyle, ".otf")
+        } catch (exception: RuntimeException) {
+            null
+        }
+    }
+
+    /**
+     * Called by renderer to resolve image file references in &lt;image&gt; elements.
+     *
+     * @param filename the filename as provided in the xlink:href attribute of a &lt;image&gt; element.
+     * @return an Android Bitmap object, or null if the image could not be found.
+     */
+    override fun resolveImage(filename: String?): Bitmap {
+        return super.resolveImage(filename)
+    }
+
+    /**
+     * Called by the parser to resolve CSS stylesheet file references in &lt;?xml-stylesheet?&gt;
+     *
+     * @param url the URL of the CSS file as it appears in the SVG file.
+     * @return a AndroidSVG CSSStyleSheet object, or null if the stylesheet could not be found.
+     */
+    override fun resolveCSSStyleSheet(url: String?): String {
+        return super.resolveCSSStyleSheet(url)
+    }
+
+    /**
+     * Called by renderer to determine whether a particular format is supported.  In particular,
+     * this method is used in &lt;switch&gt; elements when processing {@code requiredFormats}
+     * conditionals.
+     *
+     * @param mimeType A MIME type (such as "image/jpeg").
+     * @return true if your {@code resolveImage()} implementation supports this file format.
+     */
+    override fun isFormatSupported(mimeType: String?): Boolean {
+        return super.isFormatSupported(mimeType)
+    }
+
+    private fun getTypeface(
+        fontFamily: String?,
+        fontWeight: Int,
+        fontStyle: String?,
+        typefaceType: String
+    ): Typeface {
+        val style = fontStyle?.let { s ->
+            return@let when {
+                s.contains(ITALIC) -> ITALIC
+                s.contains(OBLIQUE) -> OBLIQUE
+                else -> ""
+            }
+        } ?: ""
+
+        val weight = when (fontWeight) {
+            700 -> BOLD
+            else -> ""
+        }
+
+        val fileName = if (style.isNotEmpty() && weight.isEmpty()) {
+            "$fontFamily-$style"
+        } else if (style.isEmpty() && weight.isNotEmpty()) {
+            "$fontFamily-$weight"
+        } else if (style.isNotEmpty() && weight.isNotEmpty()) {
+            "$fontFamily-$style$weight"
+        } else {
+            "$fontFamily"
+        }
+
+        return when (typefaceType) {
+            ".ttf" -> Typeface.createFromAsset(assetManager, "$fileName.ttf")
+            ".otf" -> Typeface.createFromAsset(assetManager, "$fileName.otf")
+            else -> Typeface.createFromAsset(assetManager, "$fileName.ttf")
+        }
+    }
+
+    private companion object {
+        const val BOLD = "Bold"
+        const val ITALIC = "Italic"
+        const val OBLIQUE = "Oblique"
```

Comment:
```suggestion
        private const val BOLD = "Bold"
        private const val ITALIC = "Italic"
        private const val OBLIQUE = "Oblique"
```

Reply:
The companion object is `private itself`. Hence, this is not required.
cc @Guardiola31337

Reply:
Nah, if you look at the decompiled code you'd notice that constants are ultimately `public` if not explicitly `private` 👀 

```
public class MapboxExternalFileResolver extends SVGExternalFileResolver {
   private final AssetManager assetManager;
   @NotNull
   public static final String BOLD = "Bold";
   @NotNull
   public static final String ITALIC = "Italic";
   @NotNull
   public static final String OBLIQUE = "Oblique";
```

vs

```
public class MapboxExternalFileResolver extends SVGExternalFileResolver {
   private final AssetManager assetManager;
   private static final String BOLD = "Bold";
   private static final String ITALIC = "Italic";
   private static final String OBLIQUE = "Oblique";
```